ONSdigital/eq-author
# Add date format property

### What is the context of this PR?
Implements functionality required to specify the date format of a date answer. 

### How to review 
- Run Tests
- Ensure feature works as per https://trello.com/c/bsxNpqSy/461-date-format-picker-ui-api

### Dependencies
API: https://github.com/ONSdigital/eq-author-api/pull/105
